src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivity.test.js

```diff
@@ -25,8 +27,13 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivity',
         created_by: { name: 'Jane Doe', id: 10 },
         currentUser,
         description: { message: 'test' },
+        file_version: {
+            id: '456',
+            version_number: '2',
```

Comment:
This field won't be present on a "new" annotation. Is that okay?

Reply:
I think that should be okay for now. It's a corner case that might come up with real-time updates but we're not there yet